core/exception/status_spec.rb

```diff
@@ -1,5 +1,9 @@
 require_relative '../../spec_helper'
 
 describe "SystemExit#status" do
-  it "needs to be reviewed for spec completeness"
+  it "returns the exit status" do
+    -> { exit }.should raise_error(SystemExit) { |e|
```

Comment:
It would be useful to pass a non-default exit status here, for instance 1 or 42.